src/main/reactjs/src/redux/selectors/clerkTranslator.ts

```diff
@@ -64,49 +90,123 @@ export const selectFilteredSelectedTranslators = createSelector(
 
 // Helpers
```

Comment:
Olisko helpers-metodit hyvä siirtää luokan alimmaisiksi `trimAndLowerCase` kanssa?

Reply:
Siellähän ne on. Ainakin siitä näkökulmasta että kaikki tuosta rivistä alaspäin on vain kyseisen tiedoston sisäisiä funkkareita; exportatut ovat ylempänä.